python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -52,8 +53,8 @@ def setDefaults(self):
 
     def validate(self):
         ForcedPhotImageTask.ConfigClass.validate(self)
-        if (self.measurement.doReplaceWithNoise and self.footprintDatasetName is not None
-                and self.references.removePatchOverlaps):
+        if (self.measurement.doReplaceWithNoise and self.footprintDatasetName is not None and
+                self.references.removePatchOverlaps):
```

Comment:
I'm confused by this. I see that you're squelching a `flake8` warning, but you actually seem to be going against the [PEP8 recommendation](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) (and against Knuth, which I personally think is much more significant). Is this a known issue in `flake8`? Has it been discussed elsewhere?